fixit/cli/run_rules.py

```diff
@@ -87,33 +100,38 @@ def get_formatted_reports_for_path(
     return [opts.formatter.format(rr) for rr in raw_reports]
 
 
-def main(raw_args: Sequence[str]) -> int:
-    parser = argparse.ArgumentParser(
-        description=(
-            "Validates your lint rules by running them against the specified, "
-            + "directory or file(s). This is not a substitute for unit tests, "
-            + "but it can provide additional confidence in your lint rules.\n"
-            + "If no lint rules or packages are specified, runs all lint rules "
-            + "found in the packages specified in `fixit.config.yaml`."
-        ),
-        parents=[
-            get_paths_parser(),
-            get_rules_parser(),
-            get_use_ignore_comments_parser(),
-            get_skip_ignore_byte_marker_parser(),
-            get_compact_parser(),
-            get_multiprocessing_parser(),
-        ],
-    )
-
+def _add_arguments(parser: argparse.ArgumentParser) -> None:
+    """All required arguments for `run_rules`"""
     parser.add_argument(
         "--cache-timeout",
         type=int,
         help="Timeout (seconds) for metadata cache fetching. Default is 2 seconds.",
         default=2,
     )
```

Comment:
This is duplicated to `get_metadata_cache_parser`, can you change to use `get_metadata_cache_parser`?
https://github.com/Instagram/Fixit/blob/18365db024258817f7f30006886c14ce8f07bbd5/fixit/cli/args.py#L174-L182

Reply:
ok

Reply:
Done